src/collapse/bpftrace.rs

```diff
@@ -1,20 +1,21 @@
+//! Module containting the **bpftrace** implementation of [`Frontend`].
+//!
+//! [`Frontend`]: trait.Frontend.html
```

Comment:
At this point I've stopped including links like this since RFC 1964 [is stabilizing soon](https://github.com/rust-lang/rust/issues/43466) and will entirely obviate the need for them!

Reply:
Yah - The current way you have to do it is totally broken. I'll pull all the internal links out. Leave the external links, though (e.g. for io::Error or for the webpages of bpftrace and perf)?